src/spirc.rs

```diff
@@ -59,7 +59,20 @@ fn initial_state() -> State {
     })
 }
 
-fn initial_device_state(name: String, volume: u16) -> DeviceState {
+fn initial_device_state(name: String, volume: u16, device_type: String) -> DeviceState {
+    let device_type_id = match device_type.as_ref() {
+        "unknown" => 0,
+        "computer" => 1,
+        "tablet" => 2,
+        "smartphone" => 3,
+        "speaker" => 4,
+        "tv" => 5,
+        "avr" => 6,
+        "stb" => 7,
+        "audiodongle" => 8,
+        _ => 5,
```

Comment:
Can I suggest that we set the default to the more generic `speaker` type, as a TV isn't appropriate in most cases.